src/main/java/nextstep/subway/section/Sections.java

```diff
@@ -0,0 +1,143 @@
+package nextstep.subway.section;
+
+import nextstep.subway.station.Station;
+
+import javax.persistence.Embeddable;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.util.stream.Collectors.toList;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany
+    @JoinColumn(name = "line_id")
```

Comment:
<img width="465" alt="스크린샷 2023-07-23 오후 1 28 07" src="https://github.com/next-step/atdd-subway-path/assets/17218212/ee997f4f-3845-44b3-9a87-7351476cacda">

`Section` 저장할때마다 `update`가 발생합니다 😢 
아래와 같이 설정하면 방지할 수 있습니다. (`Section`은 항상 `Line 에 포함되기 때문에 cascade 옵션을 줘도 될 것 같아요 😄 )
```suggestion
@OneToMany(mappedBy = "line", cascade = CascadeType.PERSIST)
```